src/app/scanning/handle-registration/handle-registration.component.html

```
@@ -74,14 +74,40 @@ <h2 class="title-medium">Scan result</h2>
+      <p>{{ errorMessage(checkInMutation.error()) }}</p>
+    </div>
+  } @else if (checkInCompleted()) {
+    <div
+      class="bg-primary-container text-on-primary-container mt-4 rounded p-4"
+    >
+      <p class="title-small">Check-in recorded</p>
+    </div>
+  }
   <button
-    [disabled]="!scanResultQuery.data().allowCheckin"
+    [disabled]="
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)